src/Monomer/Widgets/Singles/Base/InputField.hs

```diff
@@ -919,7 +919,7 @@ updatePlaceholder wenv node !state !config = newState where
     & L.text . non def . L.alignH ?~ inputFieldAlignH style
     & L.text . non def . L.alignV ?~ inputFieldAlignV style
   text = _ifcPlaceholder config
-  fitText = fitTextToSize fontMgr pstyle Ellipsis MultiLine KeepSpaces Nothing
+  fitText = fitTextToSize fontMgr pstyle Ellipsis MultiLine KeepSpaces OnSpaces Nothing
```

Comment:
Just to be clear, I think it's fine if we ignore the new text attribute here since it's a single line field